expan/core/experiment.py

```diff
@@ -114,6 +111,10 @@ def analyze_statistical_test(self, test, test_method='fixed_horizon', **worker_a
         # run the test method
         test_statistics = worker(x=treatment_data, y=control_data)
         test_result.result = test_statistics
+
+        # remove data from the result test metadata
+        if not include_data:
```

Comment:
Why?

Reply:
because data can be huge, we just don't want it to be present in the results. 

Here include_data is sort of "hidden" parameter, I need it to only be used by analyze_statistical_test_suite. When we run analyze_statistical_test_suite with include_data=True it means it will not delete data because we need those data when we call analyze_statistical_test again in line 170. We delete data for suites in line 180-181. When we call only analyze_statistical_test it will delete data for 1 test because include_data=False.